Sources/generate-pipeline/RuleDocumentationGenerator.swift

```diff
@@ -0,0 +1,83 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
```

Comment:
Honest question: why do we need `Foundation` in our generators?

Reply:
`write(into:)` takes a `FileHandle` argument.

Reply:
So technically we could just write into a file differently, but we just don't aim to support environments where Foundation is not available yet?

Reply:
I can't think of any platforms that don't have Foundation that I'd want to run generate-pipeline on.

The rest of swift-format also uses Foundation, so I don't know why I'd want to single out generate-pipeline specifically.